src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/subscribe_button_response.cc

```diff
@@ -55,6 +56,38 @@ SubscribeButtonResponse::~SubscribeButtonResponse() {}
 
 void SubscribeButtonResponse::Run() {
   SDL_LOG_AUTO_TRACE();
+
+  int32_t corr_id = (*message_)
```

Comment:
@ychernysheva you can remove this variable and use `correlation_id()` method of the base class wherever you need

Reply:
Done in 950d7d4